hazelcast/src/test/java/com/hazelcast/test/SplitBrainTestSupport.java

```diff
@@ -445,7 +445,7 @@ public void await() {
         }
     }
 
-    protected static class MergeIntegerValuesMergePolicy implements SplitBrainMergePolicy, DataSerializable {
+    protected static class MergeIntegerValuesMergePolicy implements SplitBrainMergePolicy, SerializationServiceAware {
```

Comment:
If we need the `SerializationService`, we can simply implement the new interface `SerializationServiceAware`.